enterprise/models.py

```diff
@@ -169,6 +178,28 @@ def requests_data_sharing_consent(self):
         """
         return self.enable_data_sharing_consent and self.enforce_data_sharing_consent != self.EXTERNALLY_MANAGED
 
+    def get_course_enrollment_url(self, course_run_key):
+        """
+        Return enterprise landing page url for the given course.
+
+        Arguments:
+            course_run_key (str): The course run id for the course to be displayed.
+        Returns:
+            (str): Enterprise landing page url.
+        """
+        if configuration_helpers is None:
+            raise NotConnectedToOpenEdX(
+                _("This package must be installed in an EdX environment to look up configuration.")
+            )
+
+        return urljoin(
+            configuration_helpers.get_value('LMS_ROOT_URL', settings.LMS_ROOT_URL),
```

Comment:
I'm thinking we might want some additional logic. We're assuming here that Enterprise will be installed inside the LMS, but that may not always be the case. We may want to create an additional variable `ENTERPRISE_PUBLIC_URL_ROOT` and default to `LMS_ROOT_URL`. I don't think we need to stick that variable into `edx-platform` or `configuration` yet, though; it's a long road to IDA-ness.